lua/luasnip/util/util.lua

```diff
@@ -233,12 +233,22 @@ local function put(text, pos)
 	pos[2] = (#text > 1 and 0 or pos[2]) + #text[#text]
 end
 
--- Wrap a value in a table if it isn't one already
-local function wrap_value(value)
-	if not value or type(value) == "table" then
-		return value
+--[[ Wraps the value in a table if it's not one, makes
+  the first element an empty str if the table is empty]]
+local function to_string_table(value)
+	if not value then
+		return { "" }
 	end
-	return { value }
+	if type(value) == "string" then
+		return { table }
```

Comment:
`table` -> `value`? :D
Was just looking into why tests are failing and imagine my surprise when a string was wrapped to a table somehow containing functions xD

Reply:
Don't program if you are sleepy my friend :facepalm: